docs-src/prod-readiness-context/data-encryption.md

```diff
@@ -35,6 +35,7 @@ User code <--> Payload Converter <--> Payload Codec <--> Wire <--> Temporal Serv
 ```
 
 You can run your Payload Codec with a [Codec Server](/concepts/what-is-a-codec-server) and use the Codec Server endpoints in Web UI and tctl to decode your encrypted Payload locally.
+For details on how to set up a Codec Server, see (/prod-readiness-context/how-to-set-up-codec-server).
```

Comment:
```suggestion
For details on how to set up a Codec Server, see [Codec Server setup](#how-to-set-up-codec-server).
```
I _think_ this is correct link target. See what Assembly thinks.

Reply:
I thought this link was fixed; will try your suggestion locally and see what works

Reply:
fixed!